src/gazebo_wind_plugin.cpp

```diff
@@ -118,35 +118,26 @@ void GazeboWindPlugin::OnUpdate(const common::UpdateInfo& _info) {
   ignition::math::Vector3d wind = wind_strength * wind_direction;
 
   ignition::math::Vector3d wind_gust(0, 0, 0);
-  // Calculate the wind gust force.
+  // Calculate the wind gust velocity.
   if (now >= wind_gust_start_ && now < wind_gust_end_) {
     // Get normal distribution wind gust strength
-    double wind_gust_strength = wind_gust_force_distribution_(wind_gust_force_generator_);
-    wind_gust_strength = (wind_gust_strength > wind_gust_force_max_) ? wind_gust_force_max_ : wind_gust_strength;
+    double wind_gust_strength = wind_gust_velocity_distribution_(wind_gust_velocity_generator_);
+    wind_gust_strength = (wind_gust_strength > wind_gust_velocity_max_) ? wind_gust_velocity_max_ : wind_gust_strength;
```

Comment:
Shouldn't this be clamped on both positive and negative sides?

Reply:
Done!